pkgs/development/mobile/genymotion/default.nix

```diff
@@ -10,10 +10,11 @@ let
   libPath = "${stdenv.lib.makeLibraryPath packages}";
 in
 stdenv.mkDerivation rec {
-  name = "genymotion-${version}";
+  pname = "genymotion";
   version = "2.8.0";
+  name = "${pname}-${version}";
```

Comment:
youi dont need to specify `name`, it will be generated from pname and version.

Reply:
Thanks for the heads up. I removed it.